static/js/components/ProgramCourseList.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React from 'react';
+import ProgramCourse from './ProgramCourse';
+
+
+export default class ProgramCourseList extends React.Component {
```

Comment:
it might be good to try to namespace this to the program page somehow, just because we have a `CourseListCard` already and it might get a bit confusing.

OR maybe we could move these components to a new directory, like `static/js/components/program_page` or something.